### PR TITLE
Update header share button

### PR DESCRIPTION
**Pull request by faihegberg**
Ensure header share button doesn't scroll to a non-existent section and instead behaves like all other share buttons on the location page.

### CODE REVIEW

File: src/components/NewLocationPage/LocationOverview/LocationOverview.tsx [hunk 1]
```diff
@@ -75,6 +79,13 @@ const LocationOverview: React.FC<{
       </GridContainer>
       <MobileOnly>
         <ShareButton onClickShare={onClickShare} />
+        <ShareButtons
+          eventCategory={EventCategory.ENGAGEMENT}
+          shareUrl={shareUrl}
+          shareQuote={shareQuote}
+          region={region}
+          isLocationPageHeader={true}
+        />
```
Comment (faihegberg) [inline src/components/NewLocationPage/LocationOverview/LocationOverview.tsx:88]: The first share button component needs to be replaced with the second share button. I just haven't removed the first one yet to better illustrate the issue. Currently, the second share button will not stretch in the same way as the first. @chasulin we can sync on this to resolve.

**Comment by chasulin** [comment]
almost there! last thing-

the placement of the social buttons looks slightly wonky on mobile:
Screen Shot 2021-05-26 at 10 46 37 AM

how about we center them instead:
Screen Shot 2021-05-26 at 10 48 17 AM

placing these styles on the absolutely positioned div for mobile should do it-
```
top: 1rem;
margin: 0 auto;
```

**Review by chasulin — COMMENTED** [review]
almost there! last thing-

the placement of the social buttons looks slightly wonky on mobile (first screenshot). how about we center them instead (2nd screenshot)? placing these styles on the absolutely positioned div for mobile should do it

top: 1rem;
margin: 0 auto;
(left and right set to 0)

Screen Shot 2021-05-26 at 10 46 37 AM
Screen Shot 2021-05-26 at 10 48 17 AM

**Review by chasulin — APPROVED** [review]
🚢

**Pull request merged**